tests/bluetooth/bsim_bt/bsim_test_gatt_write/prj.conf

```diff
@@ -12,3 +12,6 @@ CONFIG_BT_BUF_EVT_DISCARDABLE_SIZE=255
 CONFIG_BT_L2CAP_TX_MTU=247
 
 CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_LLCP_LOCAL_PROC_CTX_BUF_NUM=4
```

Comment:
Make `CONFIG_BT_CTLR_LLCP_LOCAL_PROC_CTX_BUF_NUM=4`  as default in Kconfig. Setting `CONFIG_BT_CTLR_ADVANCED_FEATURES=y` in projects will display build warning.

Reply:
Done